src/commands/misc/help.js

```diff
@@ -0,0 +1,34 @@
+const { EmbedBuilder } = require('@discordjs/builders')
+
+module.exports = {
+    name: 'help',
+    description: 'List of commands',
+
+    callback: (client, interaction) => {
+        //Create an embed to send to the user
+        const embed = new EmbedBuilder()
+        .setTitle('Server Commands')
+        .addFields(
```

Comment:
Indentation is a touch odd here, recommend
```suggestion
        const embed = new EmbedBuilder()
            .setTitle('Server Commands')
            .addFields(
```